ush/parsing_namelists_FV3.sh

```diff
@@ -344,6 +346,7 @@ cat >> input.nml <<EOF
   iopt_snf     = ${iopt_snf:-"4"}
   iopt_tbot    = ${iopt_tbot:-"2"}
   iopt_stc     = ${iopt_stc:-"1"}
+  iopt_trs     = ${iopt_trs:-"2"}
```

Comment:
@ChunxiZhang-NOAA I saw that the following variables were added in ufs-weather-model: 

  do_RRTMGP          = @[DO_RRTMGP]
  active_gases       = 'h2o_co2_o3_n2o_ch4_o2'
  ngases             = 6
  lw_file_gas        = 'rrtmgp-data-lw-g128-210809.nc'
  lw_file_clouds     = 'rrtmgp-cloud-optics-coeffs-lw.nc'
  sw_file_gas        = 'rrtmgp-data-sw-g112-210809.nc'
  sw_file_clouds     = 'rrtmgp-cloud-optics-coeffs-sw.nc'
  rrtmgp_nGptsSW     = 112
  rrtmgp_nGptsLW     = 128
  rrtmgp_nBandsLW    = 16
  rrtmgp_nBandsSW    = 14
  doGP_cldoptics_LUT = @[DOGP_CLDOPTICS_LUT]
  doGP_lwscat        = @[DOGP_LWSCAT]

as well as: 

  lndp_type      = @[LNDP_TYPE]
  lndp_model_type = @[LNDP_MODEL_TYPE]
  LNDP_TAU=21600,
  LNDP_LSCALE=500000,
  ISEED_LNDP=2010,
  lndp_var_list = @[LNDP_VAR_LIST]
  lndp_prt_list = @[LNDP_PRT_LIST]
  
  
  Are any of these additional variables of consequence to the prototype runs?  I believe it's okay not to include either, but as this did show up as a difference between what we are running the ufs-waether-model regression tests and the workflow I wanted to double check.

Reply:
The namelist settings for land perturbation has been included already. It is controlled by switch $DO_LAND_PERT. By default $DO_LAND_PERT=.false and the namelist settings will not be generated.